### PR TITLE
Feature/location dropdown

### PR DESCRIPTION
**Pull request by KoiOne**
image
IMG_8482
image
image

### CODE REVIEW

File: components/index/SearchResult/USGSWaterdatasetCard.tsx [hunk 1]
```diff
@@ -23,146 +23,165 @@ import Link from "next/link";
...
+        <div className="flex items-center">
+          <h6
+            style={{ cursor: "pointer" }}
+            // onClick={() => logLinkClick(dataset, index)}
+            onClick={() => toggleSample()}
+            className="text-xl font-bold tracking-tight text-gray-900 dark:text-white"
+          >
+            {dataset.title}
+          </h6>
+          <FlowTooltip content={``}>
```
Comment (xu-jenny) [inline components/index/SearchResult/USGSWaterdatasetCard.tsx:124]: If there's no content let's remove the Tooltip

File: components/index/LocationInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  locationDetails: {
+    display_name: string | null;
+  };
+};
+
+const LocationInfo: React.FC<LocationInfoProps> = ({
+  locationName,
+  locationDetails,
+}) => {
+  const parseLocation = (display_name: string | null) => {
```
Comment (xu-jenny) [inline components/index/LocationInfo.tsx:16]: I think this parseLocation logic is too weak. The string is not always guaranteed to be formatted this way

**Review by xu-jenny — CHANGES_REQUESTED** [review]

File: components/index/DatasetPane/USGSWaterDatasetPane.tsx [hunk 3]
```diff
@@ -5,53 +5,59 @@ import { post } from "@/utils/http";
...
+  const downloadCSV = async () => {
+    const now = new Date().toISOString();
+    const sevenDaysAgo = new Date(
+      Date.now() - 7 * 24 * 60 * 60 * 1000
+    ).toISOString();
+    const response = await post(
+      `${process.env.NEXT_PUBLIC_PYTHON_BACKEND_URL}/usgs_water_csv`,
+      {
+        siteId: dataset.siteId,
+        parameterCode: "00045", //dataset.matchingParamCode,
```
Comment (xu-jenny) [inline components/index/DatasetPane/USGSWaterDatasetPane.tsx:22]: Why is this hardcoded

**Comment by KoiOne** [comment]
IMG_6063
IMG_0607
image

**Review by xu-jenny — APPROVED** [review]

**Pull request merged**